### PR TITLE
[ADD] solution hw-6.4

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/core/basesyntax/Factorial.java [hunk 1]
```diff
@@ -6,6 +6,12 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number < 0) {
+            return -1;
+        }
+        if (number == 0 || number == 1) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Factorial.java]: let's make one condition here

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Factorial.java (hunk 1 above)
Comment (ghost) [reply src/main/java/core/basesyntax/Factorial.java]: done

File: src/main/java/core/basesyntax/Factorial.java [hunk 2]
```diff
@@ -6,6 +6,12 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number < 0) {
+            return -1;
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Factorial.java]: let's remove this 
`In mathematics, the factorial of a positive integer n, denoted by n!, is the product of all positive integers less than or equal to n:`

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Factorial.java (hunk 2 above)
Comment (ghost) [reply src/main/java/core/basesyntax/Factorial.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**